lang/nl/lang.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ *
+ * @author G. Uitslag <klapinklapin@gmail.com>
+ */
+$lang['template']              = '— Template';
+$lang['not_found']             = ' niet gevonden.';
+$lang['processing_template']   = '— Verwerken van de template ';
+$lang['stopped_recursion']     = ' gestopt vanwege recursieve aanroep.';
+$lang['no_permissions_view']   = '<div class="templater"> Geen toestemming om de template weer te geven </div>';
+$lang['no_such_section']       = '<div class="templater">— Er is geen sectie "%1$s" in deze pagina ';
```

Comment:
not sure why you include the html in the translated string? This is extra work to translate, while you have less complete overview of the your code because some parts are in the language files instead of in syntax.php. May I suggest to move the divs to syntax.php?

Reply:
I will see how to do this.